marlowe-playground-client/packages.dhall

```diff
@@ -158,7 +158,7 @@ let additions =
           //  { repo =
                   "https://github.com/shmish111/purescript-foreign-generic"
               , version =
-                  "purs-0.13"
+                  "fd7f49f47061f708021064e17bb11b193ab389a3"
```

Comment:
This version includes `Map`, `Tuple` and `Set` instances of `Encode` and `Decode`